docs/source/examples/workspaces/openstack-heat/os_stack.yml

```diff
@@ -2,12 +2,24 @@ heat_template_version: 2013-05-23
 
 description: Simple template to deploy a single compute instance
 
+parameters:
+  flavor:
+    type: string
+    label: Instance Type
+    description: Flavor to be used
+    default: m1.small
+  tags:
+    type: string
+    label: metadata
+    description: test tags
+    default: testinstancelinchpin
+
 resources:
   my_instance:
     type: OS::Nova::Server
     properties:
       image: cirros
-      flavor: m1.small
+      flavor: { get_param: flavor }
```

Comment:
what is `get_param`?

Reply:
@14rcole get_param is a hot template sytax when given parameters to ansible module openstack stack recognizes the parameters using get_param